Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:16-alpine3.11
```

```suggestion
FROM node:16-alpine3.14
```
Why suddenly degrade to lower version? We previously used 3.14 image.

---

I copied file from previous commmit, forgot to change the version. It's done now